src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -7,7 +7,10 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double sqrtNumber) {
```

Comment:
@RostykRosolovskyi why you have renamed this variable? is it just an ordinary number of a number with already calculated sqrt function?
All variable names should not confuse other developers